Assets/EngyneCreations/Trello/Scripts/TrelloAPI.cs

```diff
@@ -0,0 +1,313 @@
+﻿/*
+ * TrelloAPI.cs
+ * Interact directly with the Trello API using MiniJSON and uploads cards. 
+ * 
+ * Original by bfollington
+ * https://github.com/bfollington/Trello-Cards-Unity
+ * 
+ * by Adam Carballo under GPLv3 license.
+ * https://github.com/AdamEC/Unity-Trello
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+using MiniJSON;
+using UnityEngine.Networking;
+
+namespace Trello 
+{
+	public class TrelloAPI
+	{
+		private const string HASHEDKEY = "NmZiZDU4M2VmMzkxZjU1ZjFjZTgyODJlNWRmMDUxYjU=";
```

Comment:
![image](https://user-images.githubusercontent.com/5779541/158741035-b2ad76fa-98dc-4087-ad01-1e4b707c0d12.png)

https://developers.amadeus.com/blog/best-practices-api-key-storage 
Store the keys in a Env Variable. E.g. Check out how mixpanel does it.

Reply:
It looks like the MixPanel asset is just a scriptable object? Is that what you mean by an Env Variable?

Reply:
Yeah.
Its an
.asset file, so if you can make that for the api then it'll prob work

Reply:
@m4rtint Is the point of this to move the keys out of the repo, or just out of code?

Reply:
Out the code, it's a security thing and Github recognize if it's in code and flags it as a security issue